browser/scripts/application.js

```diff
@@ -173,6 +173,16 @@ Application.prototype.releaseHoverSlot = function() {
 	this.releaseHoverConnections();
 }
 
+Application.prototype.setSlotCssClasses = function(slot, slot_div) {	/* @var slot_div jQuery */
+	if (typeof slot_div == 'undefined') return false;
+	if (!slot_div) return false;
+	if (!(slot_div === this.hover_slot_div)) slot_div.removeClass('p_connecting')
```

Comment:
please split this into two lines, and use `if (slot_div !== this.hover_slot_div)` instead (better readability)